redesign.user.js

```diff
@@ -1153,17 +1171,24 @@ const isDashboard =
 const MOODLE_LANG = document.documentElement.lang.toLowerCase();
 const DARK_MODE_SELECTOR = 'html[data-darkreader-scheme="dark"]';
 
+const BETTER_MOODLE_LANG = // This is ugly. There probably is a better way to do this
+
+        GM_getValue(getSettingKey('general.language'), 'auto') === 'auto' ?
+            MOODLE_LANG
+        :   GM_getValue(getSettingKey('general.language'), 'auto');
+
```

Comment:
could an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) be a more beautiful solution?
smth like that:
```js
const BETTER_MOODLE_LANG = (() => {
    const savedLanguage = GM_getValue(getSettingKey('general.language'), 'auto');
    if (savedLanguage === 'auto') return MOODLE_LANG;
    return savedLanguage;
})();
```

Also I think I would prefer this next to `MOODLE_LANG` initialization (in between `MOODLE_LANG` and `DARK_MODE_SELECTOR`)